bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -27,14 +27,23 @@ futures-signals = { version = "0.3.30", default-features = false }
 futures-util = { version = "0.3.17", default-features = false }
 # FIXME: we currently can't feature flag anything in the api.udl, therefore we must enforce sliding-sync being exposed here..
 # see https://github.com/matrix-org/matrix-rust-sdk/issues/1014
-matrix-sdk = { path = "../../crates/matrix-sdk", features = ["anyhow", "experimental-timeline", "markdown", "sliding-sync", "socks"], version = "0.6.0" }
 once_cell = "1.10.0"
 sanitize-filename-reader-friendly = "2.2.1"
 serde_json = { version = "1" }
 thiserror = "1.0.30"
 tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
 tokio-stream = "0.1.8"
-tracing = { workspace = true }
-tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 uniffi = { workspace = true }
 uniffi_macros = { workspace = true }
+
+
+[target.'cfg(target_os = "android")'.dependencies]
+tracing = { version = "0.1.29", default-features = false, features = ["log"] }
+android_logger = "0.11"
+log-panics = { version = "2", features = ["with-backtrace"]}
+matrix-sdk = { path = "../../crates/matrix-sdk", default-features = false, features = ["anyhow", "experimental-timeline", "e2e-encryption", "sled", "markdown", "sliding-sync", "socks", "rustls-tls"], version = "0.6.0" }
```

Comment:
Hmm.. shouldn't iOS also have `e2e-encryption` and `sled`? 🤔

Reply:
It's enabled by default, so you don't have to, it's just that for android we have to use rustls-tls and disable default

Reply:
Right, got it! 👍